front/src/common-app/components/footer/footer.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { css } from 'emotion';
+import { theme } from 'core/theme';
+
+export const footer = css`
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  min-width: 320px;
+`;
+
+export const imgFooter = css`
+  max-width: 160px;
+  padding: 2% 0;
+
+  /* ${theme.breakpoints.down('md')} {
+    max-width: 100px;
+  } */
```

Comment:
Commented code (lechuguitas), if the comment must remain add reason why if not remove it (a comment like that does not help, another developer could doubt if it's something just not in use or if it's something that should remain there, or something important

Reply:
resolved in new commit